diff --git a/arduino-core/src/cc/arduino/contributions/packages/HostDependentDownloadableContribution.java b/arduino-core/src/cc/arduino/contributions/packages/HostDependentDownloadableContribution.java
index 93a3420..e63f332 100644
--- a/arduino-core/src/cc/arduino/contributions/packages/HostDependentDownloadableContribution.java
+++ b/arduino-core/src/cc/arduino/contributions/packages/HostDependentDownloadableContribution.java
@@ -50,17 +50,7 @@ public abstract class HostDependentDownloadableContribution extends Downloadable
     String host = getHost();
 
     if (osName.contains("Linux")) {
-      if (osArch.equals("arm")) {
-        // Raspberry PI, BBB or other ARM based host
-
-        // PI: "arm-linux-gnueabihf"
-        // Arch-linux on PI2: "armv7l-unknown-linux-gnueabihf"
-        // Raspbian on PI2: "arm-linux-gnueabihf"
-        // Ubuntu Mate on PI2: "arm-linux-gnueabihf"
-        // Debian 7.9 on BBB: "arm-linux-gnueabihf"
-        // Raspbian on PI Zero: "arm-linux-gnueabihf"
-        return host.matches("arm.*-linux-gnueabihf");
-      } else if (osArch.contains("amd64")) {
+      if (osArch.contains("amd64")) {
         return host.matches("x86_64-.*linux-gnu");
       } else {
         return host.matches("i[3456]86-.*linux-gnu");
