diff --git a/app/src/processing/app/UpdateCheck.java b/app/src/processing/app/UpdateCheck.java
index 39c5550..96c59af 100644
--- a/app/src/processing/app/UpdateCheck.java
+++ b/app/src/processing/app/UpdateCheck.java
@@ -100,26 +100,6 @@ public class UpdateCheck implements Runnable {
       }
       PreferencesData.set("update.last", String.valueOf(now));
 
-      String prompt =
-        tr("A new version of Arduino is available,\n" +
-          "would you like to visit the Arduino download page?");
-        
-      if (base.activeEditor != null) {
-        if (latest > BaseNoGui.REVISION) {
-          Object[] options = { tr("Yes"), tr("No") };
-          int result = JOptionPane.showOptionDialog(base.activeEditor,
-                                                    prompt,
-                                                    tr("Update"),
-                                                    JOptionPane.YES_NO_OPTION,
-                                                    JOptionPane.QUESTION_MESSAGE,
-                                                    null,
-                                                    options,
-                                                    options[0]);
-          if (result == JOptionPane.YES_OPTION) {
-            Base.openURL(tr("http://www.arduino.cc/en/Main/Software"));
-          }
-        }
-      }
     } catch (Exception e) {
       //e.printStackTrace();
       //System.err.println("Error while trying to check for an update.");
